src/data/featured-data.jsx

```diff
@@ -0,0 +1,35 @@
+import faker from 'faker';
+faker.seed(123);
+const featuredData = [...Array(20)].map((item) => ({
+	id: faker.random.uuid(),
+	name: faker.commerce.productName(),
+	image: faker.random.image(),
+	inStock: faker.random.boolean(),
+	type: faker.random.arrayElement(['Consoles', 'Games', 'Accessories ']),
+	hasOffer: faker.random.boolean(),
+	offer: faker.random.arrayElement(['Save $20', '10% off', 'Buy one get one free', 'Stock clearance sale ']),
+
+	//! COMMENTED FOR FUTURE
```

Comment:
Everything looks fine remove commented code

Reply:
That is for future reference